### PR TITLE
Update styles of MoviesPage

### PR DESCRIPTION
**Pull request by RayRedGoose**
This PR contains next changes: 
- added background to App component
- updated header styles
- added styles for movies container 
- added styles for small movie card
- added loading image 
- added router for redirection to single movie page

Note: I moved images folder inside src folder, because I can't get access to it when it outside src folder.

MoviesPage
Screen Shot 2019-12-17 at 10 18 52 PM

Loading image animation
Screen Recording 2019-12-17 at 10 19 PM

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './App.scss';
 import MoviesContainer from '../../containers/MoviesContainer/MoviesContainer';
 import Header from '../Header/Header.js';
-import { Route, Redirect } from 'react-router-dom';
+import { Route } from 'react-router-dom';
```
Comment (cammac60) [inline src/components/App/App.js:5]: Thanks for fixing this, I meant to remove that before making my last PR.

File: src/components/LoadingImage/LoadingImage.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.loading {
+  display: flex;
+  align-items: center;
+  margin: auto;
+  position: relative;
+  .light {
```
Comment (cammac60) [inline src/components/LoadingImage/LoadingImage.scss:6]: Very cool animation, nice work!

**Review by cammac60 — APPROVED** [review]
Really great work. Loving the styles changes, especially the animation!

File: src/components/LoadingImage/LoadingImage.scss (hunk 2 above)
Comment (RayRedGoose) [reply src/components/LoadingImage/LoadingImage.scss:6]: Thanks!

**Pull request merged**